docs/discovery-item.v3.html

```diff
@@ -0,0 +1,418 @@
+    <div class="nypl-item-details">
+      <h1>
+      <!-- titleDisplay -->
+      Chinese Muslims and the global Ummah : Islamic revival and ethnic identity among the Hui of Qinghai Province
+      </h1>
+
+      <!-- bordered div -->
+      <!-- <div class="nypl-item-info"> -->
+        <!-- <p> -->
+        <!-- materialType / issuance (language ? + " in " + language) -->
+        <!-- <span class="nypl-item-media">Text / Monograph</span> in Catalán
+        </p> -->
+
+        <!-- <p> -->
+        <!-- placeOfPublication publisher[with link] then if (dateStartYear == dateEndYear ? dateStartYear : dateStartYear + " - " + dateEndYear (if dateEndYear == 9999, replace dateEndYear value with "present"))[with link] -->
+        <!-- Barcelona : [Tarragona] : <a href="http://dev-discovery.nypl.org/search?q=publisher%3A%22Waterways%20Project%20of%20Ten%20Penny%20Players%2C%20Inc.%2C%22" title="Make a new search for Publisher: Columna ; Diputació de Tarragona, Museu d'Art Modern,">Columna ; Diputació de Tarragona, Museu d'Art Modern,</a> <a href="http://dev-discovery.nypl.org/search?q=date%3A1995" title="Make a new search for Year Published: 1995">1995</a> -->
+        <!-- </p> -->
+
+        <!-- extent then dimensions -->
+        <!-- <p>
+        x, 221 pages ; 25 cm
+        </p> -->
+
+        <!-- <p>SASB - Microforms Rm 119 (<a href="http://www.nypl.org/locations/schwarzman">how to get there?</a>)</p> -->
+
+        <!-- <p class="nypl-item-use">In-library use (Advance request required)</p> -->
+      <!-- </div> -->
+      <!-- end bordered div -->
+
+
+    <!-- end item detail div -->
+    <!-- column for extra item info -->
+
+
+      <!-- <div class="nypl-item-details"> -->
+        <dl>
+          <dt>format</dt>
+          <dd>Text / Monograph in Catalán</dd>
+
+          <dt>Author</dt>
+          <dd><a href="#">Stewart, Alexander Blair, author.</a></dd>
+
+          <dt>Contributors</dt>
+          <dd><!-- comma separated links --><a href="#">Susanna, Alex.</a>, <a href="#">Galià i Adell, Narcís, 1925-</a></dd>
+
+          <dt>description</dt>
+          <dd>x, 221 pages ; 25 cm</dd>
+
+          <dt>bibliography</dt>
+          <dd>Barcelona : [Tarragona] : <a href="http://dev-discovery.nypl.org/search?q=publisher%3A%22Waterways%20Project%20of%20Ten%20Penny%20Players%2C%20Inc.%2C%22" title="Make a new search for Publisher: Columna ; Diputació de Tarragona, Museu d'Art Modern,">Columna ; Diputació de Tarragona, Museu d'Art Modern,</a> <a href="http://dev-discovery.nypl.org/search?q=date%3A1995" title="Make a new search for Year Published: 1995">1995</a></dd>
+
+          <dt>Alternative titles</dt>
+          <dd>
+            <ul>
+              <li>Chinese Muslims and the global Ummah : Islamic revival and ethnic identity among the Hui of Qinghai Province</li>
+              <li>Chinese Muslims and the global Ummah :</li>
+            </ul>
+          </dd>
+        </dl>
+        <aside class="place-hold-control inline">
```

Comment:
Indeed. the documentation can be interpreted in a way that supports my pattern: 
http://html5doctor.com/understanding-aside/
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside
and cue the actual spec:
https://www.w3.org/TR/html5/sections.html#the-aside-element

at any rate, @seanredmond asked that I approach it differntly, see that in the v.4 version, embedding the table in the `<dl>`